### PR TITLE
Add data for Web Serial API.

### PR DESCRIPTION
**Pull request by jpmedley**
https://www.chromestatus.com/feature/6577673212002304

### CODE REVIEW

File: api/Navigator.json [hunk 1]
```diff
@@ -2382,6 +2382,54 @@
           }
         }
       },
+      "serial": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/serviceWorker",
```
Comment (foolip) [inline api/Navigator.json:2387]: This URL and that in `WorkerNavigator` needs to be updated.

Comment (jpmedley) [reply api/Navigator.json]: Fixed. PTAL

File: api/WorkerNavigator.json [hunk 2]
```diff
@@ -144,6 +144,54 @@
           }
         }
       },
+      "serial": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/serviceWorker",
```
Comment (foolip) [inline api/WorkerNavigator.json:149]: This URL also needs updating.

**Comment by foolip** [comment]
@jpmedley I tried verifying this with collector results but then noticed it's Chrome 89 which hasn't been released yet. Would you like to merge this before Chrome 89 is released?

File: api/WorkerNavigator.json (hunk 2 above)
Comment (jpmedley) [reply api/WorkerNavigator.json]: Fixed. PTAL

**Pull request merged**